CHANGELOG.md

```diff
@@ -40,6 +48,18 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 
 - BC-137 - fix bug with display of Blue Bar
 
+## [26.9.3] - 2021-09-10
+
+### Fixed
+
+- BC-239 - normalize students birth date
```

Comment:
are the changes in this file correct?!

Reply:
yes ..the basic hotfix is go out as 9.3. but is must also merge to the already exist release 26.10.x and by merge this i make a misstake and we must solve this misstake 26.10.2...not nice but seldom.